samples/charts/financial-chart/annotations/Program.cs

```diff
@@ -25,6 +25,9 @@ public static async Task Main(string[] args)
             // registering Infragistics Blazor
             builder.Services.AddScoped(typeof(IInfragisticsBlazor), typeof(InfragisticsBlazor));
 
+            builder.Services.AddScoped(typeof(Samples.Services.StocksUtility));
+            builder.Services.AddScoped(typeof(Samples.Services.Stock));
```

Comment:
you should not add the `StocksUtility` and `Stock` classes here because they are not services